yarn.lock

```diff
@@ -15,20 +15,25 @@
     "@jridgewell/gen-mapping" "^0.3.5"
     "@jridgewell/trace-mapping" "^0.3.24"
 
```

Comment:
Shoud yarn.lock changes be included into this PR? 
CC @BamMironov

Reply:
I will remove it.